docs/DeveloperGuide.md

```diff
@@ -233,6 +233,12 @@ From the notes in the sequence diagram above, for every attribute in the instrum
 prompt to get user to provide information for that attribute. This is done through a series of methods in
 the `TextUi` class.
 
+### Edit instrument feature
+<img src="images/EditInstrumentSequenceDiagram.png" width="600"/>
+<img src="images/EditRefrence1.png" width="600"/>
```

Comment:
<img width="468" alt="ss4" src="https://user-images.githubusercontent.com/60163283/140518096-329d7518-3bc8-4de1-9dc2-b95911d9caa8.PNG">
getAllParams, same for below